src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/avoidescapedunicodecharacters/Example5.java

```diff
@@ -0,0 +1,34 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="AvoidEscapedUnicodeCharacters">
+      <property name="allowNonPrintableEscapes" value="true"/>
+    </module>
+  </module>
+</module>
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.avoidescapedunicodecharacters;
+
+// xdoc section -- start
```

Comment:
Move this special comment after `public class Example5 {`
To reduce amount of code lines we print in xdoc files.

Move `xdoc section -- end` to one line up also.

Apply same for all other examples.

Reply:
> Move this special comment after `public class Example5 {` To reduce amount of code lines we print in xdoc files.
> 
> Move `xdoc section -- end` to one line up also.
> 
> Apply same for all other examples.

This will also show indentation error. Doing the way i did it this time also maintain the uniformity in between other checks .